ironfish-cli/bin/run

```diff
@@ -7,6 +7,11 @@ if (process.platform !== 'win32') {
   require('segfault-handler').registerHandler('segfault.log')
 }
 
+if(process.versions.node.split('.')[0] !== '16') {
+  console.log('NodeJS version ' +  process.versions.node + ' is not compatible. Must have node v16.x installed. You can install using nvm: https://github.com/nvm-sh/nvm')
```

Comment:
Can we just link to node 16? NVM might be too hard for non technical people.

Reply:
Yea good call